index.css

```diff
@@ -752,4 +852,39 @@ textarea::placeholder {
     align-self: center;
     margin-bottom: 50px;
   }
+
+  .pop-up-menu {
+    width: 70vw;
+    height: 90vh;
+  }
+
+  .pop-up-menu-wrapper {
+    display: flex;
+  }
```

Comment:
Great design team🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the Figma design provided. I would recommend you fix your desktop popup image as it appears distorted and blurred at the moment.
- Your design
![Your-desktop-image](https://github.com/Samuelwanza/portfolio_setup_and_mobile_first/assets/63917725/93586af0-392e-41f6-98be-1a1de2b0e26c)
- Given design
![Given-image](https://github.com/Samuelwanza/portfolio_setup_and_mobile_first/assets/63917725/6eae7b47-5b1b-4130-85c8-e1d319c5e981)